### DISCUSSION TITLE
problem with login on github

### DISCUSSION THREAD

**Discussion by AntonioPasan07** [vercel/community#3970] [Account Issue] [answered]
How can I solve this problem, it tells me account is blocked when starting with github in vercel but I have the application in github and vercel permissions too

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble logging in. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Reply by AntonioPasan07** [reply]
Hello, good afternoon, it is already resolved, you can contact me with support and we will solve it for you, thank you very much 😉
